datadog.go

```diff
@@ -122,13 +125,17 @@ func sanitizeMetricName(namespace string, v *view.View) string {
 }
 
 // viewSignature creates the view signature with custom namespace
-func viewSignature(namespace string, v *view.View) string {
-	var buf strings.Builder
-	buf.WriteString(sanitizeMetricName(namespace, v))
-	for _, k := range v.TagKeys {
-		buf.WriteString("_" + k.Name())
+func viewSignature(namespace string, disableTagMetricName bool, v *view.View) string {
+	if !disableTagMetricName {
```

Comment:
This is an "alpha" library. So it's fine to break things. Is there any value to keep the old behaviour? Would anyone want that? If no, let's break things :)

Reply:
Users might be currently using the metric with the old name, so, renaming will break their monitors/dashboards.

The only value is not breaking metrics for current user.

But if breaking things is “ok”, we can probably drop the old behaviour, but I don’t know how many people is using the library.

I didn't knew this lib was in alpha. Should we indicate that somewhere if that's the case?

Reply:
Anything that doesn't have releases greater than `v1.0.0` is considered alpha in the Go ecosystem. We don't even have `v0.*` releases here, maybe we should? 

But you're absolutely right, we should mention this in the README.

> But if breaking things is “ok”, we can probably drop the old behaviour, but I don’t know how many people is using the library.

Well, we don't know that. But clearly the repository has adoption because there's activity here. The question comes down to: what is the correct way of doing it? Is it the new way? If yes, let's drop the old one. If both are correct, then we should keep both.

I'm thinking that the whole reason of being in alpha is allowing a library to take shape and reach a stable form that is of good use to all. It's unpleasant to break things, but if it's for the better, we should. I will let you decide this.